### PR TITLE
fix: Implemented the web view component  to the sample and corrected private attributes

### PR DESCRIPTION
**Pull request by luisgustavozup**
#242 
**Problem**
Attributes classes components are private.
Outdated sample, missing webview component .

**Solution**
Adding the web view component to the sample.
Changed private to public methods.

### CODE REVIEW

File: backend/sample/core/src/main/kotlin/br/com/zup/beagle/sample/builder/WebViewScreenBuilder.kt [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+ * limitations under the License.
+ */
+
+package br.com.zup.beagle.sample.builder
+
+import br.com.zup.beagle.action.ShowNativeDialog
+import br.com.zup.beagle.ext.applyFlex
+import br.com.zup.beagle.sample.constants.PATH_URL_WEB_VIEW_ENDPOINT
+import br.com.zup.beagle.widget.core.Flex
+import br.com.zup.beagle.widget.layout.*
```
Comment (botbeaglezup) [inline backend/sample/core/src/main/kotlin/br/com/zup/beagle/sample/builder/WebViewScreenBuilder.kt:23]: <table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/9894660a4c854201ccd617d503abff8d36962f4f/backend/sample/core/src/main/kotlin/br/com/zup/beagle/sample/builder/WebViewScreenBuilder.kt#L23"></span>br.com.zup.beagle.widget.layout.* is a wildcard import. Replace it with fully qualified imports.</td>
    </tr>
  </tbody>
</table>

File: backend/sample/micronaut/src/main/kotlin/br/com/zup/beagle/sample/micronaut/controller/ScreenController.kt [hunk 2]
```diff
@@ -46,6 +46,8 @@ import br.com.zup.beagle.sample.constants.SCREEN_SCROLL_VIEW_ENDPOINT
 import br.com.zup.beagle.sample.constants.SCREEN_TAB_VIEW_ENDPOINT
 import br.com.zup.beagle.sample.constants.SCREEN_TEXT_ENDPOINT
 import br.com.zup.beagle.sample.constants.SCREEN_TOUCHABLE_ENDPOINT
+import br.com.zup.beagle.sample.constants.SCREEN_WEB_VIEW_ENDPOINT
+import br.com.zup.beagle.sample.micronaut.service.*
```
Comment (botbeaglezup) [inline backend/sample/micronaut/src/main/kotlin/br/com/zup/beagle/sample/micronaut/controller/ScreenController.kt:50]: <table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/9894660a4c854201ccd617d503abff8d36962f4f/backend/sample/micronaut/src/main/kotlin/br/com/zup/beagle/sample/micronaut/controller/ScreenController.kt#L50"></span>br.com.zup.beagle.sample.micronaut.service.* is a wildcard import. Replace it with fully qualified imports.</td>
    </tr>
  </tbody>
</table>

File: backend/sample/core/src/main/kotlin/br/com/zup/beagle/sample/builder/WebViewScreenBuilder.kt (hunk 1 above)
Comment (botbeaglezup) [inline backend/sample/core/src/main/kotlin/br/com/zup/beagle/sample/builder/WebViewScreenBuilder.kt:23]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/9894660a4c854201ccd617d503abff8d36962f4f/backend/sample/core/src/main/kotlin/br/com/zup/beagle/sample/builder/WebViewScreenBuilder.kt#L23"></span>br.com.zup.beagle.widget.layout.* is a wildcard import. Replace it with fully qualified imports.</td>
    </tr>
  </tbody>
</table>

File: backend/sample/micronaut/src/main/kotlin/br/com/zup/beagle/sample/micronaut/controller/ScreenController.kt (hunk 2 above)
Comment (botbeaglezup) [inline backend/sample/micronaut/src/main/kotlin/br/com/zup/beagle/sample/micronaut/controller/ScreenController.kt:50]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/9894660a4c854201ccd617d503abff8d36962f4f/backend/sample/micronaut/src/main/kotlin/br/com/zup/beagle/sample/micronaut/controller/ScreenController.kt#L50"></span>br.com.zup.beagle.sample.micronaut.service.* is a wildcard import. Replace it with fully qualified imports.</td>
    </tr>
  </tbody>
</table>

File: backend/sample/core/src/main/kotlin/br/com/zup/beagle/sample/builder/ComponentScreenBuilder.kt [hunk 3]
```diff
@@ -74,12 +74,13 @@ object ComponentScreenBuilder : ScreenBuilder {
...
                 createMenu("Touchable", SCREEN_TOUCHABLE_ENDPOINT),
-                createMenu("Analytics", SCREEN_ANALYTICS_ENDPOINT)
+                createMenu("Analytics", SCREEN_ANALYTICS_ENDPOINT),
+                createMenu("Web View", SCREEN_WEB_VIEW_ENDPOINT)
             )
         )
     )
 
-    private fun createMenu(text: String, path: String) = Button(
+    private fun createMenu(text: String, path: String) = br.com.zup.beagle.widget.ui.Button(
```
Comment (LG95) [inline backend/sample/core/src/main/kotlin/br/com/zup/beagle/sample/builder/ComponentScreenBuilder.kt:83]: Please import this.

**Pull request merged**